microsoft-edge/toc.yml

```diff
@@ -1782,6 +1789,9 @@
         - name: Microsoft Edge IDE integration
           href: ./visual-studio-code/ide-integration.md
 
+      # - name: Use DevTools as an IDE
```

Comment:
toc > Microsoft Edge IDE integration > add jump page "Use DevTools as in-browser IDE (Sources tool Workspace tab)"

though doesn't work well for all frameworks per findings by @captainbrosset that are gathered in Workspaces article.  @leahmsft 

* **Edit and save files in a workspace (Sources tool Workspace tab)**
   * `/devtools-guide-chromium/workspaces/index.md`
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/workspaces/index?branch=pr-en-us-3432
   * External preview: https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/workspace/microsoft-edge/devtools-guide-chromium/workspaces/index.md
   * Before/live: https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/workspaces/